### PR TITLE
Update copyright year to 2017

### PR DESCRIPTION
**Pull request by jbakosi**
See script/update_copyright.sh for how the copyright was updated.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Copyright (c) 2016, Los Alamos National Security, LLC
+Copyright (c) 2017, Los Alamos National Security, LLC
```
Comment (junghans) [inline LICENSE]: Wouldn't it be 2016-2017?

Comment (jbakosi) [inline LICENSE]: I have no idea what is better.

Comment (jbakosi) [inline LICENSE]: I guess we should do the range as that is consistent with the copyright in the individual files.

Comment (junghans) [inline LICENSE]: My thinking was, 2016 is when LANL took over and hence the copyright holds from 2016-now (2017).

Comment (jbakosi) [inline LICENSE]: Yeah, makes sense. I will fix it.

**Pull request merged**